src/reactive-handler.ts

```diff
@@ -98,13 +98,18 @@ export class ReactiveProxyHandler {
         /* No op */
     }
     has(shadowTarget: ReactiveMembraneShadowTarget, key: PropertyKey): boolean {
-        const { originalTarget, membrane: { valueObserved } } = this;
+        const { originalTarget, membrane } = this;
+        const { valueObserved } = membrane;
         valueObserved(originalTarget, key);
-        return key in originalTarget;
+        // since key is never going to be undefined, and magicPropertyKey might be undefined
+        // we can simply compare them as the second part of the condition.
+        return key in originalTarget || key === membrane.magicPropertyKey;
     }
     ownKeys(shadowTarget: ReactiveMembraneShadowTarget): string[] {
         const { originalTarget } = this;
-        return ArrayConcat.call(getOwnPropertyNames(originalTarget), getOwnPropertySymbols(originalTarget));
+        const { magicPropertyKey } = this.membrane;
+        const keys = magicPropertyKey ? [magicPropertyKey] : [];
+        return ArrayConcat.call(keys, getOwnPropertyNames(originalTarget), getOwnPropertySymbols(originalTarget));
```

Comment:
You can call `Reflect.ownKeys(originalTarget)` to get them without doing two separate calls of `getOwnPropertyNames` and `getOwnPropertySymbols`.

Reply:
This lib is written in ways to support IE11, so we stick to the basis.

Reply:
Instead of concat you can use something like `ArrayPush` since you don't need to create a new array:
```js
ArrayPush.apply(keys, getOwnPropertyNames(originalTarget));
ArrayPush.apply(keys, getOwnPropertySymbols(originalTarget));
```

Reply:
will do that in another PR...

Reply:
Cool, things LGTM

Reply:
will do